presto-main/src/main/java/com/facebook/presto/server/security/ActiveDirectoryBinder.java

```diff
@@ -35,14 +26,13 @@
     private static final Logger LOG = Logger.get(ActiveDirectoryBinder.class);
     private final String activeDirectoryDomain;
     private final String userObjectClass;
-    private final Optional<String> baseDistinguishedName;
+    private final String searchInput = "sAMAccountName";
```

Comment:
why do you have this as a field here, instead of just having it locally in the searchFilter method?